Documentation/flexvolume.md

```diff
@@ -81,6 +81,11 @@ If you want to use Rook with an already provisioned Tectonic cluster, please ref
 
 Continue with [configuring the FlexVolume path](#configuring-the-flexvolume-path) to configure Rook to use the FlexVolume path.
 
+### OpenStack Magnum
+OpenStack Magnum using Atomic uses a non-standard FlexVolume plugin directory:  `/var/lib/kubelet/volumeplugins`
```

Comment:
What about changing this sentence to `OpenStack Magnum is using Atomic, which uses a non-standard FlexVolume plugin directory at: ...` ?

Reply:
Done!